src/Interpreters/ErrorLog.cpp

```diff
@@ -61,6 +67,18 @@ ColumnsDescription ErrorLogElement::getColumnsDescription()
                 std::make_shared<DataTypeUInt8>(),
                 parseQuery(codec_parser, "(ZSTD(1))", 0, DBMS_DEFAULT_MAX_PARSER_DEPTH, DBMS_DEFAULT_MAX_PARSER_BACKTRACKS),
                 "Remote exception (i.e. received during one of the distributed queries)."
+            },
+        {
+                "last_error_trace",
```

Comment:
`system.trace_log` also contains `symbols` and `lines` columns, so let's add them here as well.